pkg/autoscaler/autoscaler.go

```diff
@@ -218,6 +222,6 @@ func (a *Autoscaler) currentSpec() DeciderSpec {
 	return a.deciderSpec
 }
 
-func (a *Autoscaler) podCountLimited(desiredPodCount, currentPodCount float64) int32 {
-	return int32(math.Min(desiredPodCount, a.deciderSpec.MaxScaleUpRate*currentPodCount))
+func podCountLimited(desiredPodCount, currentPodCount, maxScaleUpRate float64) int32 {
```

Comment:
I debate the usefulness of this method. Since if it is inlined then the comment at the point of invocation can be removed.

Reply:
I agree! Removed and inlined.